sdv/multi_table/base.py

```diff
@@ -162,6 +163,29 @@ def set_address_columns(self, table_name, column_names, anonymization_level='ful
         self._validate_table_name(table_name)
         self._table_synthesizers[table_name].set_address_columns(column_names, anonymization_level)
 
+    def _detect_single_table_cag(self, patterns):
+        """Filter out single table CAG patterns from the list of patterns.
+
+        Args:
+            patterns (list):
+                A list of CAG patterns to filter.
+        """
+        idx_single_table = None
+        seen_single_table = False
+        for idx, pattern in enumerate(patterns):
```

Comment:
I know you're going to fix this in the end but might as well call them constraints